### PR TITLE
Added parent attribute and toggle click emit

### PR DESCRIPTION
**Pull request by TimDirks**

### CODE REVIEW

File: src/components/Collapse/Collapse.vue [hunk 1]
```diff
@@ -24,5 +26,14 @@ import {idProps} from '@/composables/useId';
...
 defineProps({
     ...idProps,
+    // Should be a valid query selector.
+    parentSelector: {
+        type: String,
+        default: null,
+    },
 });
+
+const emit = defineEmits(['click:toggle']);
```
Comment (barisbora) [inline src/components/Collapse/Collapse.vue:36]:
```suggestion
const emit = defineEmits<{(event: 'click:toggle'): void}>();
```

**Review by barisbora — APPROVED** [review]

File: src/components/Collapse/Collapse.test.ts [hunk 2]
```diff
@@ -7,4 +8,12 @@ describe('template', () => {
     });
 
     componentSlotRenderTest(Collapse);
+
+    it('emits event on click collapse toggle', async() => {
+        const wrapper = shallowMount(Collapse);
+
+        wrapper.findComponent(Collapse).vm.$emit('click:toggle');
+
+        expect(wrapper.emitted('click:toggle')).toBeTruthy();
```
Comment (thomasowow) [inline src/components/Collapse/Collapse.test.ts:17]: This test will never ever fail. It is like giving the answers of a test to a student. You need to test that the `click` event is resulting in the `click:toggle` event.

File: src/components/Collapse/Collapse.vue [hunk 3]
```diff
@@ -3,6 +3,7 @@
         class="d-inline-block"
         data-bs-toggle="collapse"
         :data-bs-target="`#${id}`"
+        @click="clickToggle"
```
Comment (thomasowow) [inline src/components/Collapse/Collapse.vue:6]:
```suggestion
        @click="emit('click:toggle')"
```

**Review by thomasowow — CHANGES_REQUESTED** [review]
Your commit messages will not result in new versions to be released.
Read this: https://github.com/semantic-release/semantic-release#commit-message-format

**Review by thomasowow — APPROVED** [review]

**Pull request merged**